mysql-test/suite/innodb/t/doublewrite.test

```diff
@@ -119,6 +120,35 @@ let SEARCH_PATTERN=InnoDB: Recovered page \[page id: space=[1-9][0-9]*, page num
 XA ROLLBACK 'x';
 check table t1;
 select f1, f2 from t1;
+
+connect (dml,localhost,root,,);
+XA START 'x';
+insert into t1 values (6, repeat('%', @@innodb_page_size/2));
+XA END 'x';
+XA PREPARE 'x';
+disconnect dml;
+connection default;
+
+flush table t1 for export;
+
+let $restart_parameters=;
+let $shutdown_timeout=0;
+--source include/shutdown_mysqld.inc
```

Comment:
Please specify `CREATE TABLE t1 … STATS_PERSISTENT=0;` so that we will not get any unexpected log checkpoints here.

Use `no_checkpoint_start.inc` and `no_checkpoint_end.inc` around the critical section, and specify a non-existing table in the cleanup action so that the test will fail in an obvious way if an unexpected checkpoint occurs.